crates/bevy_tasks/src/task_pool.rs

```diff
@@ -415,4 +441,27 @@ mod tests {
         assert!(!thread_check_failed.load(Ordering::Acquire));
         assert_eq!(count.load(Ordering::Acquire), 200);
     }
+
+    #[test]
+    fn test_spawn_pollable() {
+        let transform_fn = |i| i + 1;
+
+        let pool = TaskPool::new();
+        let nums: Range<u8> = 0..10;
+
+        let pollable_tasks = nums
+            .clone()
+            .into_iter()
+            .map(|i| pool.spawn_pollable(async move { transform_fn(i) }))
+            .collect::<Vec<_>>();
+
+        std::thread::sleep(Duration::from_secs_f32(1.0 / 30.0));
```

Comment:
Is the sleep here to wait for the tasks to finish? This should be something more robust like waiting for the tasks to finish and then checking them.

Reply:
Any idea?